helm/kyverno-policies/values.yaml

```diff
@@ -1,2 +1,5 @@
 kyverno-policies:
   podSecurityStandard: restricted
+  customLabels:
+    application.giantswarm.io/team: shield
+    policy.giantswarm.io/giantswarm-exempt: "false"
```

Comment:
This is only needed when a policy doesn't have a team label :smile:

Reply:
Ah, ack, then I ditch it.